src/mapping/impl/BasisFunctions.hpp

```diff
@@ -139,24 +139,22 @@ class VolumeSplines {
  */
 class Gaussian {
 public:
-  Gaussian(const double shape, const double supportRadius = std::numeric_limits<double>::infinity())
-      : _shape(shape),
-        _supportRadius(supportRadius)
+  Gaussian(const double shape)
+      : _shape(shape)
   {
     PRECICE_CHECK(math::greater(_shape, 0.0),
                   "Shape parameter for radial-basis-function gaussian has to be larger than zero!");
-    PRECICE_CHECK(math::greater(_supportRadius, 0.0),
-                  "Support radius for radial-basis-function gaussian has to be larger than zero!");
 
-    _deltaY          = hasCompactSupport() ? evaluate(supportRadius) : 0;
-    double threshold = std::sqrt(-std::log(cutoffThreshold)) / shape;
-    _supportRadius   = std::min(supportRadius, threshold);
+    /// Below that value the function is supposed to be zero. Defines the support radius if not explicitely given
+    constexpr double cutoffThreshold = 1e-9;
+
+    _supportRadius = std::sqrt(-std::log(cutoffThreshold)) / shape;
```

Comment:
This would still require the user to calculate the shape parameter before hand from the desired support radius? We should have something like 

```
Gaussian(const double supportRadius)

shape = std::sqrt(-std::log(cutoffThreshold)) / _supportRadius;
```
Or am I wrong?

Reply:
> This would still require the user to calculate the shape parameter before hand from the desired support radius?

Yes. This PR is just a simplification of the rbf code. It does not change anything in the actual program. Currently, passing a support radius to the Gaussian is not used and not tested.

> We should have something like

I think #765 is a more appropriate place to discuss this.